src/App.js

```diff
@@ -10,16 +10,27 @@ class App extends Component {
   constructor() {
     super();
     this.state = {
-      movies: movieData.movies
+      movies: movieData.movies,
+      movieShow: []
     }
   }
 
+  goToIndex = () => {
+    this.setState({movieShow: []})
+  }
+
+  toggleMovie = (id) => {
+    const movie = this.state.movies.find(movie => movie.id === id)
+    this.state.movieShow.length ? this.setState({movieShow: []}) :this.setState({movieShow: [movie]})
```

Comment:
if it's just going to be a single movie, then I don't think it makes sense to have it in an array. I'd just set it to the movie object